est/003.est

```diff
@@ -0,0 +1,32 @@
+date: 17-2-2016
+author: Maksim Vakhnik
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: GitHub integration
+  3: Facebook integration
+  4: Google+ integration
+  5: User login
+  6: User logout
+  7: User account
+  8: Email notifications
+  9: Starting bouts
+  10: Inviting others to bouts
+  11: Adding attachments to bouts
+  12: Markdown support
+  13: Unread messages support
```

Comment:
@2686747 I am not sure this is part of the scope, Or else, it meant something else.